packages/salesforcedx-utils-vscode/src/helpers/utils.ts

```diff
@@ -14,11 +14,16 @@ export function isNullOrUndefined(object: any): object is null | undefined {
 }
 
 export function extractJsonObject(str: string): any {
-
   const jsonString = str.substring(
     str.indexOf('{'),
     str.lastIndexOf('}') + 1
   );
 
   return JSON.parse(jsonString);
 }
+
+export async function asyncFilter<T>(arr: T[], predicate: any) {
```

Comment:
Added so `filter()` can be called with async function within it.

Reply:
Love it - great addition!

Reply:
It isn't being used anywhere yet right? We should add a few units tests to both test and document usage via test. That can def be a follow on PR

Reply:
found it in the orgUtil